src/containers/MapContainer/container.js

```diff
@@ -1,8 +1,5 @@
 import { connect } from 'react-redux';
```

Comment:
Map & MapContainer - what's the difference? Needs more meaningful naming. 'Container' word is excess in Container folder